src/Server.ts

```diff
@@ -0,0 +1,33 @@
+import * as express from 'express';
+import Iconfig from './config/iconfig';
+
+class Server {
+    private app: express.Express;
+    constructor(private config: Iconfig) {
+        this.app = express();
+    }
+    bootstrap() {
+        console.log('inside bootstrap');
+        this.setupRoutes();
+        return this;
+    }
+    run = () => {
+        const { app, config: { port } } = this;
+        app.listen(port, (err) => {
+            if (err) {
+                throw err;
+            }
+            console.log('app is running successfully');
```

Comment:
fetch the env currently, you are working on and print the port on which this app is running through environment variables.

Reply:
Done